src/components/BrainfuckEngine.vue

```diff
@@ -106,19 +106,43 @@
 
     <form @submit.prevent="input" style="margin: 12px 0px">
       <b-field>
-        <b-input v-model="inputText" :disabled="!isRunning"> </b-input>
+        <b-input v-model="inputText" :disabled="isRunning && inPause">
```

Comment:
```suggestion
        <b-input v-model="inputText" :disabled="!isRunning">
```

I'd like to this change to be reverted because input texts before running are deleted in the `run` method:
https://github.com/arkark/brainfuck-online-simulator/blob/42f1ee76164f13e8e356c0bc329b89902783802a/src/components/BrainfuckEngine.vue#L286